### ISSUE TITLE
app looks

### ISSUE THREAD

**Issue by M1Aston** [Sparker0i/Weather#45]
A lot of info combined, which makes things look a bit cluttered\messy. I think some simple modifications would make it a bit cleaner.
- Sunset\sunrise, change to a 24-hour time instead of using AM PM.
07:35  19:14     is cleaner than      07:35 AM   07:14 PM

- Replace the ""wind symbol"" above the wind speed with the arrow (direction) that is now next to the temperature. Makes more sense and look cleaner. Please, a slightly bigger arrow. Also , I don't understand points in the direction the wind is coming from. Normally, it's the other way around. Arrow points in wind directions.
Please allow wind speed km/h instead of m/s.

- In the listed days (bottom of the screen) there is a large gap between the highest\lowest temperatures. Why? Looks odd. A smaller space or a dash looks better, I think.

**Comment by Sparker0i** [comment]
>     * Sunset\sunrise, change to a 24-hour time instead of using AM PM.
>       07:35  19:14     is cleaner than      07:35 AM   07:14 PM
Agreed. Will happen in the next version

>     * Replace the ""wind symbol"" above the wind speed with the arrow (direction) that is now next to the temperature. Makes more sense and look cleaner. Please, a slightly bigger arrow. Also , I don't understand points in the direction the wind is coming from. Normally, it's the other way around. Arrow points in wind directions.
>       Please allow wind speed km/h instead of m/s.
Makes sense. Happening in the next version

>     * In the listed days (bottom of the screen) there is a large gap between the highest\lowest temperatures. Why? Looks odd. A smaller space or a dash looks better, I think.
It was designed to be that way. Not changing.

**Comment by Starcommander** [comment]
I agree with M1Aston, that the gap between min and max temp looks odd.
It looks more, as the min temp of first day belongs to the second day, and so on.
A little bit confusing imho.

**Comment by Sparker0i** [comment]
> It looks more, as the min temp of first day belongs to the second day, and so on.
If that would have been the case, the design of the 10-day view would have been incoherent and inconsistent

I liked that look a lot and that is not changing
